<ui version="4.0" >
 <class>TTCutAVCutDlg</class>
 <widget class="QDialog" name="TTCutAVCutDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TTCut - Video Cut Options</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabCommon" >
      <attribute name="title" >
       <string>Common</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="avcutdialog.qrc" >
        <normaloff>:/pixmaps/pixmaps/settings_18.xpm</normaloff>:/pixmaps/pixmaps/settings_18.xpm</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="gbFreeDriveSpace" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Free Drive Space</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_3" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Used</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLabel" name="label_4" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Free</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Used %</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="laPath1" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="laSize1" >
            <property name="text" >
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="laUsed1" >
            <property name="text" >
             <string>Used</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLabel" name="laFree1" >
            <property name="text" >
             <string>Free</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" >
           <widget class="QLabel" name="laUsedPercent1" >
            <property name="text" >
             <string>Free</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="laPath2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="laSize2" >
            <property name="text" >
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="laUsed2" >
            <property name="text" >
             <string>Used</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="QLabel" name="laFree2" >
            <property name="text" >
             <string>Free</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4" >
           <widget class="QLabel" name="laUsedPercent2" >
            <property name="text" >
             <string>Free</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="gbCutOptions" >
         <property name="title" >
          <string>Cut Options</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="cbEndCode" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Write Sequence End Code</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="cbMaxBitrate" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Write max bitrate to first sequence</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="gbOutput" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title" >
          <string>Cut Output Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <property name="spacing" >
           <number>12</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="laOutputFile" >
            <property name="text" >
             <string>Output file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QLineEdit" name="leOutputFile" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="laOutputPath" >
            <property name="text" >
             <string>Output path</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="leOutputPath" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QPushButton" name="btnDirOpen" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>54</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>54</width>
              <height>32</height>
             </size>
            </property>
            <property name="sizeIncrement" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="avcutdialog.qrc" >
              <normaloff>:/pixmaps/pixmaps/fileopen_24.xpm</normaloff>:/pixmaps/pixmaps/fileopen_24.xpm</iconset>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>leOutputPath</zorder>
         <zorder>laOutputPath</zorder>
         <zorder>leOutputFile</zorder>
         <zorder>laOutputFile</zorder>
         <zorder>btnDirOpen</zorder>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEncoding" >
      <attribute name="title" >
       <string>Encoding</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="avcutdialog.qrc" >
        <normaloff>:/pixmaps/pixmaps/exec.png</normaloff>:/pixmaps/pixmaps/exec.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="TTCutSettingsEncoder" name="encodingPage" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMuxing" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Muxing</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="avcutdialog.qrc" >
        <normaloff>:/pixmaps/pixmaps/filter.png</normaloff>:/pixmaps/pixmaps/filter.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="TTCutSettingsMuxer" name="muxingPage" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabChapters" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <attribute name="title" >
       <string>Chapters</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="avcutdialog.qrc" >
        <normaloff>:/pixmaps/pixmaps/chapter_18.xpm</normaloff>:/pixmaps/pixmaps/chapter_18.xpm</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="TTCutSettingsChapter" name="chaptersPage" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="title" >
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTCutSettingsMuxer</class>
   <extends>QGroupBox</extends>
   <header>gui/ttcutsettingsmuxer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutSettingsChapter</class>
   <extends>QGroupBox</extends>
   <header>gui/ttcutsettingschapter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutSettingsEncoder</class>
   <extends>QGroupBox</extends>
   <header>gui/ttcutsettingsencoder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="avcutdialog.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TTCutAVCutDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TTCutAVCutDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
